### PR TITLE
[Auto Update Docker] AUD-demisto/auto_update_docker_staging_branch_6

### PR DESCRIPTION
**Pull request by samuelFain**
## Related Issues
relates: [CIAC-9308](https://jira-dc.paloaltonetworks.com/browse/CIAC-9308)

## Description
Staging branch name: `AUD-demisto/auto_update_docker_staging_branch_6`

Updated docker images:
>demisto/cloudshare,demisto/bs4-py3,demisto/unzip,demisto/python-phash,demisto/jq

### CODE REVIEW

File: Packs/XSOARLabUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.yml [hunk 1]
```diff
@@ -5,13 +5,13 @@ args:
...
-  Extracts the index.zip and filters new packs since the last run.
-  Builds the slack message for new packs.
+comment: 'Extracts the index.zip and filters new packs since the last run.
+
+  Builds the slack message for new packs.'
 commonfields:
   id: BuildSlackBlocksFromIndex
   version: -1
-dockerimage: demisto/unzip:1.0.0.19258
+dockerimage: demisto/unzip:1.0.0.100283
```
Comment (JudahSchwartz) [inline Packs/XSOARLabUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.yml:14]: @talzich can you look this over?

**Review by JudahSchwartz — APPROVED** [review]
great! asked @talzich to check one out

**Review by JudahSchwartz — APPROVED** [review]

File: Packs/XSOARLabUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.yml (hunk 1 above)
Comment (talzich) [reply Packs/XSOARLabUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.yml:14]: @JudahSchwartz It looks like the script only uses Python's built-in ZipFile library and not the one from CommonScripts. Also, the dockerfile of unzip, and the one it starts from, does not contain the problamatic `pyminizip` library.

Comment (samuelFain) [reply Packs/XSOARLabUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.yml:14]: @JudahSchwartz @talzich Are we good to merge?

Comment (talzich) [reply Packs/XSOARLabUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.yml:14]: I don't see any issues

**Pull request merged**